tests/test_vcf_readcount_annotator.py

```diff
@@ -37,6 +37,7 @@ def test_error_more_than_one_sample_with_wrong_sample_name(self):
             vcf_readcount_annotator.main(command)
         self.assertTrue('does not contain a sample column for sample nonexistent_sample.' in str(context.exception))
 
+    '''
```

Comment:
I really would prefer if we could update this test with expected data instead. We probably want to replace the current test data with Zach’s data since those are actual files instead of manually created ones.